content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,38 @@
+---
+title: IMDSv2 Enablement By Default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+# IMDSv2 Default Behavior and Host Metadata Enhancement
+
+Agent v7.64.0 enables IMDSv2 by default for enhanced security and improved metadata handling. The host metadata payload contains a field, `legacy-resolution-hostname`, that captures the AWS instance hostname using legacy resolution logic (when IMDSv2 was disabled) and is only included when needed.
```

Comment:
It's not necessary to go into the technical details of how we do things, just that the change is being made and what the effects may be

Reply:
We also should link to AWS' deprecation notices here